contracts/base/AuctioneerManaged.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.4.11;
+
+contract AuctioneerManaged {
+    // auctioneer has the power to manage some variables
+    address public auctioneer;
+    
+    function AuctioneerManaged () public {
```

Comment:
DX upgrade has no initializer, but AuctioneerManaged has one? and its inherited.

Do you know what is happening there under the hood?

Reply:
I think the constructors of all the parents are initialized. Since there's no arguments, you can do it in an implicit way. Otherwise you have to be explicit in the child's constructor.

Also if there's no constructor, it creates a empty constructor for you.

https://solidity.readthedocs.io/en/develop/contracts.html#arguments-for-base-constructors

Reply:
kk

However, it seems like the auctioneer is also set here:
function setupDutchExchange(
        TokenFRT _frtToken,
        TokenOWL _owlToken,
        address _auctioneer, 
        address _ethToken,
        PriceOracleInterface _ethUSDOracle,
        uint _thresholdNewTokenPair,
        uint _thresholdNewAuction
    )

Maybe you want to use only one of these options and not both.

Reply:
You are totally right! I thought there was a constructor also in the original contract but there wasn't

I'll remove it